.github/workflows/pipeline.yml

```diff
@@ -55,10 +55,11 @@ jobs:
         run: chmod +x mvnw
 
       - name: Lint (Formatter Validate)
-        run: ./mvnw -B clean package formatter:validate impsort:check -Dquarkus.quinoa.run-tests=false
+        run: ./mvnw -B formatter:validate impsort:check -DskipTests
+
 
       - name: Build with unit and integration tests
-        run: ./mvnw -B -ntp verify
+        run: ./mvnw -B -ntp verify -Dquarkus.quinoa.run-tests=false
```

Comment:
I would like to execute tests using the quinoa, but we can do it in another pull request.